OpenEdXMobile/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -75,7 +79,7 @@ private void init() {
 
         // initialize Fabric
         if (config.getFabricConfig().isEnabled() && !BuildConfig.DEBUG) {
-            Fabric.with(this, new CrashlyticsCore(), new Answers());
```

Comment:
I don't view it as frequently as I do with Crashlytics it gives a great overall picture of our app.